tests/integration/examples/test_notebooks_gpu.py

```diff
@@ -608,15 +656,24 @@ def test_cornac_bivae_integration(
 @pytest.mark.gpu
 @pytest.mark.integration
 @pytest.mark.parametrize(
-    "data_dir, num_epochs, batch_size, expected_values, seed",
+    "data_dir, num_epochs, batch_size, model_name, expected_values, seed",
     [
         (
             os.path.join("tests", "recsys_data", "RecSys", "SASRec-tf2", "data"),
             1,
             128,
+            "sasrec",
             {"ndcg@10": 0.2626, "Hit@10": 0.4244},
             42,
-        )
+        ),
+        (
+            os.path.join("tests", "recsys_data", "RecSys", "SASRec-tf2", "data"),
+            1,
+            128,
+            "ssept",
+            {"ndcg@10": 0.2626, "Hit@10": 0.4244},
```

Comment:
@aeroabir this is great, the only detail to check is, are you sure that both algos get the same NDCG and HIT?

Reply:
@miguelgfierro the algorithms generate different results but the current TOL and ABS_TOL take care of the differences.